api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1414,22 +1414,20 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.isRetired());
```

Comment:
That seems to make sense, now that validation is kicking in, you might need that extra text data